notebooks/samples/tensorflow/prediction_auto-scaling/ai_platform_prediction_auto-scaling.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>I wasn't aware of these metric target parameters for CPUs and GPUs. Can you provide a bit of context about what this configuration means? I'm assuming it means "scale up nodes when <code>CPU_USAGE</code> and <code>GPU_DUTY_CYCLE</code> hit 80%" but it would be nice to see that clarified outright. Preferably, this would be in text above the code cell rather than as a code comment in the same cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I cleaned up that section with a markdown explanation. Thanks!